Uno.Gallery/Uno.Gallery.UITest/Given_CheckBox.cs

```diff
@@ -6,34 +6,28 @@
 using NUnit.Framework;
 using Uno.UITest.Helpers.Queries;
 using Uno.UITests.Helpers;
-using Query = System.Func<Uno.UITest.IAppQuery, Uno.UITest.IAppQuery>;
 
 namespace Uno.Gallery.UITests
 {
-    public class Given_CheckBox : TestBase
-    {
-        [Test]
-        public void When_ClickMaterial()
-        {
-            if(AppInitializer.GetLocalPlatform() == Platform.Android)
+	public class Given_CheckBox : TestBase
+	{
+		[Test]
+		public void When_ClickMaterial()
+		{
+			if (AppInitializer.GetLocalPlatform() == Platform.Android)
 			{
-                Assert.Ignore();
+				Assert.Ignore();
 			}
```

Comment:
Can you try removing this and see if it helps ?

Reply:
I did, and reverted it because it is still failing. Do you remember why you disabled it in the first place?